.circleci/config.yml

```diff
@@ -74,7 +74,7 @@ jobs:
           command: rustc --version > rust.version
       - restore_cache:
           keys:
-            - v7-release-cache-{{ checksum "./rust.version" }}-{{ checksum "./Cargo.lock" }}
+            - v9-release-cache-{{ checksum "./rust.version" }}-{{ checksum "./Cargo.lock" }}
```

Comment:
Did you mean to co-mingle the v7/v8 & v9 cache versions?

Reply:
I was trying to normalize the vX numbers.  I might replace these custom per-job save/restore with the save/restore commands at top.

Reply:
I switch all of the tasks to using the shared save/restore cache commands.